fe/fe-core/src/main/java/org/apache/doris/catalog/CatalogRecycleBin.java

```diff
@@ -70,6 +70,56 @@ public CatalogRecycleBin() {
         idToRecycleTime = Maps.newHashMap();
     }
 
+    public synchronized boolean allTabletsInRecycledStatus(List<Long> backendTabletIds) {
```

Comment:
The `backendTabletIds` may contains millions of ids, which may cause a long run.

Reply:
Thanks, as we've just discussed, i will add a threshold to control this check and add some logs.